app.py

```diff
@@ -106,12 +106,18 @@ def static_url(filename):
 
         return url + '?v=' + file_hash.hexdigest()[:7]
 
+    if 'openid' in flask.session:
```

Comment:
@tbille Not sure if that's the best way to expose signed in user to all templates.

@Lukewh Also, as far as I remember we shouldn't be showing user name that comes from SSO, but the one from store API, right? So we would need to get it here?

Reply:
@bartaz correct, we should be using the name provided by the store API

Reply:
@tbille Any suggestions how to do that? I guess we don't want to call user name from API every time any page with user is loaded? I guess instead (or together) with storing 'openid' in session we should load and keep store user name in session once user logs in?

Reply:
we could store some information in the session? We will have to make that when we logout we delete everything.

Reply:
mmm but we will have a problem. If a users logsin and goes to directly to the market page (without passing via the acocunt page) we will need a wrapper that would get the username from the API

Reply:
@bartaz @Lukewh I did a small PR to fix this problem https://github.com/canonical-websites/snapcraft.io/pull/500